scio-google-cloud-platform/src/it/scala/com/spotify/scio/bigquery/BigQueryIT.scala

```diff
@@ -58,15 +58,23 @@ class BigQueryIT extends AnyFlatSpec with Matchers {
       shakespeare("thy", 30, "kinglear", 1600)
     )
     val expected = Seq(wordCount("i", 10), wordCount("thou", 20), wordCount("thy", 30))
+  }
 
+  "MockBigQuery" should "support mock data" in {
     val mbq = MockBigQuery()
-    mbq.mockTable(tableRef).withData(inData)
-    mbq.queryResult(legacyQuery) should contain theSameElementsAs expected
-    mbq.queryResult(sqlQuery) should contain theSameElementsAs expected
+    mbq.mockTable(tableRef).withData(MockBQData.inData)
+    mbq.queryResult(legacyQuery) should contain theSameElementsAs MockBQData.expected
+    mbq.queryResult(sqlQuery) should contain theSameElementsAs MockBQData.expected
   }
 
-  // see https://cloud.google.com/bigquery/docs/querying-wildcard-tables
-  it should "support wildcard tables" in {
+  "MockBigQuery" should "support mock data with custom staging dataset" in {
```

Comment:
use `it` if the main test group does not change

```suggestion
  it should "support mock data with custom staging dataset" in {
```